python/dgl/distributed/kvstore.py

```diff
@@ -98,7 +98,8 @@ def process_request(self, server_state):
         if kv_store.data_store.__contains__(self.name) is False:
             raise RuntimeError("KVServer Cannot find data tensor with name: %s" % self.name)
         local_id = kv_store.part_policy[self.name].to_local(self.id_tensor)
```

Comment:
Should we add a check:` kv_store.push_handlers.__contains__(self.name) is False:`